src/sat.h

```diff
@@ -24,6 +25,8 @@
 #include "sattypes.h"
 #include "worker.h"
 
+constexpr char kProcessError[] = "sat-process-error";
```

Comment:
inline constexpr absl::string_view if you can use it: go/totw/140#an-inline-constexpr-variable

Reply:
There's no implcit string conversion in OSS land so if I did that (and I did try it already 🙂) then i would need to explicity convert it tot a string or char array, so we lose any of the benefit that we woudl get. We sometimes use constexpr char arrays in internal oss code for the same reason.